src/ListTabulator/index.ts

```diff
@@ -735,64 +833,90 @@ export default class ListTabulator {
     if (!currentItem) {
       return;
     }
+
+    /**
+     * Check that the item has potential parent
```

Comment:
is this comment actual? how parent is related to previous sibling ?

Reply:
previous sibling is potential parent in case of adding tab
(after adding tab current item would be moved to previous sibling child list)